crates/router/src/connector/stripe/transformers.rs

```diff
@@ -201,13 +218,53 @@ pub struct StripeBankRedirectData {
     pub bank_specific_data: Option<BankSpecificData>,
 }
 
+#[derive(Debug, Eq, PartialEq, Serialize)]
+#[serde(untagged)]
```

Comment:
Why we need untagged here the parent struct which use the this have method type we can infer that and serialize.